.gitignore

```diff
@@ -2,3 +2,4 @@
 /bin/configlet
 /pnpm-lock.yaml
 /yarn.lock
+/.idea
```

Comment:
In this particular repo we want to keep out user-specific ignores from the ignore file. Please define this in a personal / global gitignore. <https://www.atlassian.com/git/tutorials/saving-changes/gitignore#personal-git-ignore-rules>

Reply:
Thanks... It's done.  ✅